reddit.js

```diff
@@ -1,5 +1,37 @@
 var reddit = require("./data.json")
 
 // Write your solutions below
+// Question 1
+var titles = reddit.data.children.forEach(function(titleName){
```

Comment:
Here, a more appropriate name for your parameter would be child, since your forEach function is iterating over each child of the children array.